### ISSUE TITLE
Completion for `extern "rust-intrinsic" fn`s in std

### ISSUE THREAD

**Issue by kojole** [racer-rust/racer#878]
Completion for `extern "rust-intrinsic" fn`s in std fails.
e.g. `std::mem::transmute`, `std::ptr::copy`, `std::ptr::copy_nonoverlapping`

```
$ racer complete std::mem::t
MATCH transmute_copy,832,14,/Users/kojole/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/src/libstd/../libcore/mem.rs,Function,pub unsafe fn transmute_copy<T, U>(src: &T) -> U

$ racer complete std::ptr::c
```

### Versions
- rustc 1.27.0 (3eda71b00 2018-06-19)
- racer 2.0.14

**Comment by kngwyu** [comment]
Thanks for reporting!
It looks a bit harder than #877

**Comment by kngwyu** [comment]
Closed via #895
